src/inmanta/module.py

```diff
@@ -3079,7 +3079,7 @@ def _get_fq_mod_name_for_py_file(self, py_file: str, plugin_dir: str, mod_name:
     def execute_command(self, cmd: str) -> None:
         print(f"executing {cmd} on {self.name} in {self._path}")
         print("=" * 10)
-        subprocess.call(cmd, shell=True, cwd=self._path)
+        subprocess.call(cmd, cwd=self._path)
         print("=" * 10)
```

Comment:
I saw another usage of this argument, this method is only used in a `do` method in `ModuleTool`. Can it be taken out?

Reply:
cc @arnaudsjs

Reply:
I don't think we can remove it on the `do()` method, because it would be a breaking change on the `do` command. But I don't think the `do` command is used in practice. Maybe it would be a good idea to just drop that command entirely. Let's create a ticket to discuss this in the planning meeting.

Reply:
Ticket has been created: https://github.com/inmanta/inmanta-core/issues/7913